torchxrayvision/models.py

```diff
@@ -345,6 +348,26 @@ def fix_resolution(x, resolution: int, model: nn.Module):
             warning_log[hash(model)] = True
         return model.upsample(x)
     return x
+
+def warn_normalization(x):
+    """Check normalization of input and warn if possibly wrong. When 
+    processing an image that may likely not have the correct 
+    normalization we can issue a warning. But running min and max on 
+    every image/batch is costly so we only do it on the first image/batch.
+    """
+    
+    # Only run this check on the first image so we don't hurt performance.
+    if not "norm_check" in warning_log:
+        x_min = x.min()
+        x_max = x.max()
+        if not torch.logical_and(x_min < -255, x_max > 255):
+            print(f'Warning: Input image does not appear to normalized correctly. The input image has the range [{x_min:.2f},{x_max:.2f}] which doesn\'t seem to be in the [-1024,1024] range. This warning may be wrong though. Only the first image is tested and we are only use a heurstic in an attempt to save a user from using the wrong normalization.')
```

Comment:
Should you also raise a warning if there are values lower/greater than -1024 / 1024?

Reply:
Oh ya we should also do that. I'll add it.